keyboards/mwstudio/mw80/info.json

```diff
@@ -0,0 +1,119 @@
+{

```

Comment:
This file needs to use Unix line endings (LF), not windows (CRLF). You must do this locally; the GitHub web editor cannot change it.
![image](https://user-images.githubusercontent.com/4781841/194792117-b239d7b8-41fe-49c6-bdd7-acbb3f4d4fac.png)